frontend/src/util/commonInterfaces.ts

```diff
@@ -0,0 +1,58 @@
+export interface CommentThreadsResult {
```

Comment:
VS Code extension suggestion -"JSON to TS", great for interface creation, but doesn't appear to support pointer resolution. 👎